### ISSUE TITLE
Case-Insensitivity issue in integration/vendor

### ISSUE THREAD

**Issue by marco-jantke** [containous/traefik#1822]
### Do you want to request a *feature* or report a *bug*?

bug

### What did you do?

Checkout latest master version.

### What did you expect to see?

Nothing unexpected.

### What did you see instead?

In the integration/vendor directory we have logrus twice. Once with `Sirupsen` in the path and once with `sirupsen`. You can only have one of those folders on a case-insensitive file-system and therefore git shows you either:

`modified:   vendor/github.com/sirupsen/logrus/entry.go`

or 

`modified:   vendor/github.com/Sirupsen/logrus/entry.go`

### Output of `traefik version`: (_What version of Traefik are you using?_)

Its latest master version.

```
traefik version
Version:      dev
Codename:     cheddar
Go version:   go1.8.3
Built:        I don't remember exactly
OS/Arch:      darwin/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

I am working on Traefik with a Mac with a case-insensitive file-system. `File System Personality:  Journaled HFS+`

The change was introduced at this point: https://github.com/containous/traefik/pull/1813/files#diff-969bc80135afc51d6e41fc01504e83f1R139

**Comment by ldez** [comment]
We already know this issue.
It's not a Traefik bug. We cannot fixed mistakes made by sirupen.

Currently, we work on cleaning glide. Stay connect! 😃

**Comment by marco-jantke** [comment]
Ok great. Good luck down there in the mud :)

**Comment by ldez** [comment]
I reopen only to track the subject.

**Comment by ldez** [comment]
Closed by #1823
